src/collectors/cephstats/cephstats.py

```diff
@@ -8,12 +8,38 @@
 import re
 import os
 import sys
-
 sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__)),
                                 'ceph'))
 from ceph import CephCollector
 
 
+# This is external to the CephCollector so it can be tested
+# separately.
+def process_ceph_status(output):
+    pattern = re.compile(r'\bclient io .*')
+    res = pattern.search(output)
+    if not res:
+        return {}
+    try:
+        ceph_stats = res.group()
+        number = re.compile(r'\d+')
+        rd = number.search(ceph_stats)
+        if rd:
```

Comment:
how so?  If 'rd' is False or None, the others will remain undefined (i.e. None) and the "if" checks later will not pass.